pkg/dev-env/docker-compose.yml

```diff
@@ -157,15 +241,15 @@ services:
       POSTGRES_DB: postgres
       POSTGRES_PASSWORD: postgres
 
-  ## Gamedao graph service
+  ## GameDAO graph service
   gamedao-graph-service:
     build: '../service'
     environment:
       IPFS_API_URL: "https://ipfs.infura.io:5001/api/v0"
       IPFS_CLIENT_ID: "1v04L2wj5JmI0JgKF5KztV0oN8o"
       IPFS_CLIENT_SECRET: "85547c6003abb67a6335469d1aa6a3a3"
       SERVER_PORT: 4000
-      CHAIN_RPC_URL: "ws://subzero:9944"
+      CHAIN_RPC_URL: "ws://subzero-relay-a:9944"
```

Comment:
why do you want to connect to the relay instead of the parachain ?

Reply:
Fixed, forgot to change it back coz I had some troubles setting up the nodes.